backend/drm/backend.c

```diff
@@ -89,6 +91,8 @@ static void session_signal(struct wl_listener *listener, void *data) {
 			struct wlr_drm_plane *plane = conn->crtc->cursor;
 			drm->iface->crtc_set_cursor(drm, conn->crtc,
 				(plane && plane->cursor_enabled) ? plane->cursor_bo : NULL);
+			drm->iface->crtc_move_cursor(drm, conn->crtc, conn->cursor_x,
+				conn->cursor_y);
```

Comment:
I think it does make sense to do this to prevent bouncing cursors. We basically set the plane position before enabling the cursor to make sure it doesn't show up one frame in the wrong place.